### PR TITLE
bump charts to common v3

### PR DESCRIPTION
**Pull request by onedr0p**
**Description of the change**

- bump charts using the common library to common v3
- bump each chart major version
- include `ingress.enabled=false` on each chart `values.yaml`
- include "IMPORTANT NOTE" about each app chart using the common library in each chart `values.yaml`
- set `kubeVersion: ">=1.16.0"` each app charts `Chart.yaml`
- bump docker image versions on a few charts
- fix indent consistency

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

**Comment by onedr0p** [comment]
100% this will not pass in the CI and we should merge if no issues are found on review.

File: charts/flaresolverr/Chart.yaml [hunk 1]
```diff
@@ -2,18 +2,19 @@ apiVersion: v2
...
 description: FlareSolverr is a proxy server to bypass Cloudflare protection
 name: flaresolverr
-version: 1.1.0
+version: 2.0.0
+kubeVersion: ">=1.16.0"
 keywords:
-  - flaresolverr
-  - jackett
+- flaresolverr
+- jackett
```
Comment (bjw-s) [inline charts/flaresolverr/Chart.yaml:9]: is this somehow related to Jackett? maybe add the cloudflare tag instead?

Comment (onedr0p) [reply charts/flaresolverr/Chart.yaml:9]: As far as I know it only works with jackett, i'm pretty sure it's maintained by the same people.

File: charts/samba/values.yaml [hunk 2]
```diff
@@ -22,12 +28,14 @@ service:
...
-  # SHARE1: share1;/share/samba/share1
-  # SHARE2: share2;/share/samba/share2
+ingress:
+  enabled: false
+
+probes:
+  liveness:
+    enabled: false
+  readiness:
+    enabled: false
```
Comment (bjw-s) [inline charts/samba/values.yaml:38]: This will probably need startupProbe disabled as well
